source/StarField.cpp

```diff
@@ -137,6 +137,10 @@ void StarField::Draw(const Point &pos, const Point &vel, double zoom, const Syst
 	bool isParallax = (parallaxSetting == Preferences::BackgroundParallax::FANCY ||
 						parallaxSetting == Preferences::BackgroundParallax::FAST);
 
+	// This slows down the effect of the player's zoom when they go past 0.25 to prevent tiling
+	if(baseZoom < 0.35)
+		baseZoom = baseZoom + ((0.35 - baseZoom) / 2);
```

Comment:
Is there any reason we're using this scaling in particular instead of a logarithmic curve?